content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+> TEXT TO WRITE:
+> 
+> * talk about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app")
+>    * link to kubeflow-overview for more details
+>    * (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+> * short discussion about how Kubeflow is versioned, 
+>    * components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+> 
+> * short discussion about the two "paths" for installing kubeflow "XXXXX" and "YYYYY"
```

Comment:
Proposal for overview text (edited from #2625):

> Kubeflow is an MLOps platform that aggregates a set of [core components](link) with [external add-ons](link) to orchestrate Machine Learning processes on Kubernetes.

> Kubeflow supports ML workflows from data preparation through training and deployment. It enables a scalable pipeline structure that helps ensure containerized steps with defined dependencies. Kubeflow extends Kubernetes’ ability to run independent and configurable systems with a curated set of compatible tools and frameworks specific for ML. 

Proposal for 2 paths text (edited from #2625):

> There are two options for getting up and running with Kubeflow. You can select one of the supported Kubeflow distributions from the table below, or you can roll your own version, directly from [manifests](https://github.com/kubeflow/manifests).

Reply:
I have implemented something similar to this.